daemon/config/config_unix.go

```diff
@@ -96,12 +96,29 @@ func verifyDefaultCgroupNsMode(mode string) error {
 	return nil
 }
 
+func verifyUlimits(ulimits map[string]*units.Ulimit) error {
```

Comment:
Maybe a unit test?

Reply:
@cpuguy83 I tried one (see first commit) but I couldn't test it properly.

Reply:
@cpuguy83 I created a unit test, it passes!